client/components/dashboard/dashboard-service.js

```diff
@@ -1041,7 +1044,7 @@ DashboardService.prototype.addWriter = function() {
  * @export
  */
 DashboardService.prototype.addParam = function() {
-  this.current.model.params.push(new DashboardParam());
+  this.current.model.params.push(new DashboardParam('', ''));
```

Comment:
I think that default arguments are a bit too magical for a simple data storage class like this, seems clearer to have it always contain the values provided as input. Falling back to defaults can lead to nasty bugs if it happens unintentionally.